src/foremast/elb/create_elb.py

```diff
@@ -14,6 +14,7 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 """Create ELBs for Spinnaker Pipelines."""
+import os
```

Comment:
mentioned below, but we should avoid getting environment variables this far into foremast unless really needed.